phoenicis-javafx/src/main/java/org/phoenicis/javafx/JavaFXApplication.java

```diff
@@ -19,15 +19,43 @@
 package org.phoenicis.javafx;
 
 import javafx.application.Application;
+import javafx.concurrent.Task;
+import javafx.concurrent.Worker;
+import javafx.geometry.Pos;
+import javafx.geometry.Rectangle2D;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.control.ProgressBar;
+import javafx.scene.effect.DropShadow;
 import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.Pane;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
 import javafx.scene.text.Font;
+import javafx.stage.Screen;
 import javafx.stage.Stage;
+import javafx.stage.StageStyle;
 import org.phoenicis.javafx.controller.MainController;
 import org.phoenicis.multithreading.ControlledThreadPoolExecutorServiceCloser;
+import org.phoenicis.repository.RepositoryManager;
+import org.slf4j.LoggerFactory;
 import org.springframework.context.ConfigurableApplicationContext;
 import org.springframework.context.annotation.AnnotationConfigApplicationContext;
 
+import java.util.Arrays;
+import java.util.List;
+
+import static org.phoenicis.configuration.localisation.Localisation.tr;
+
 public class JavaFXApplication extends Application {
+    private final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(JavaFXApplication.class);
+
+    private double splashWidth;
+    private double splashHeight;
+    private Pane splashLayout;
```

Comment:
Are all these properties required on an object scope?

Reply:
They are used in `init` and `showSplash`.